apps/launchpad/package.json

```diff
@@ -79,10 +80,12 @@
     "eslint-plugin-react-hooks": "^4.6.0",
     "postcss": "^8.4.38",
     "remix-flat-routes": "^0.6.4",
+    "stream-browserify": "^3.0.0",
```

Comment:
What did we use this and buffer for?

Only asking because these have historically introduced build issues with vite